webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo';
+
+// import from a file named `bar.js` located in the parent directory:
+import f3 from '..bar';
```

Comment:
All of these need a slash to indicate the directory.
Current directory uses `./filename`
Parent directory uses `../filename`